extensions/amp-mathml/1.0/component.js

```diff
@@ -0,0 +1,129 @@
+import objStr from 'obj-str';
+
+import {MessageType, deserializeMessage} from '#core/3p-frame-messaging';
+import {tryParseJson} from '#core/types/object/json';
+
+import * as Preact from '#preact';
+import {useCallback, useEffect, useMemo, useState} from '#preact';
+import {forwardRef} from '#preact/compat';
+import {useIntersectionObserver, useValueRef} from '#preact/component';
+import {ProxyIframeEmbed} from '#preact/component/3p-frame';
+import {useMergeRefs} from '#preact/utils';
+
+import {useStyles} from './component.jss';
+import {TYPE} from './utils';
+
+const ORIGIN_MATCHER = () => true;
+
+/**
+ * @param {!BentoMathmlDef.Props} props
+ * @param {{current: BentoMathmlDef.Api}} ref
+ * @return {PreactDef.Renderable}
+ */
+function BentoMathmlWithRef(props, ref) {
+  const {
+    formula,
+    inline = false,
+    title = 'MathML formula',
+    style: bentoMathmlStyles = {},
```

Comment:
```suggestion
    style: bentoMathmlStyles,
```

Does this need a default value? I think if you spread `...undefined` it's equivalent

Reply:
not necessary, i can remove it

Reply:
removed